src/messages.json

```diff
@@ -55,7 +55,8 @@
         "ProgramTextEditor.programLabel": "Program:",
         "ProgramSpeedController.slider": "Program play speed",
         "AudioFeedbackToggleSwitch.audioFeedback": "Audio feedback",
-        "Scene": "Scene",
+        "Scene.inBound": "Scene, {numColumns} by {numRows} grid with a robot character at column {xPos}, row {yPos}",
```

Comment:
I think we will need to include the direction that the character is facing also.

Please rename this message to `Scene.inBounds`.